src/layout/Tabs/Tabs.tsx

```diff
@@ -41,6 +41,11 @@ export interface TabsProps extends PropsWithChildren {
    */
   style?: React.CSSProperties;
 
+  /**
+   * The variant of the tabs.
+   */
+  variant?: 'primary' | 'secondary';
```

Comment:
We `secondary` is what we are calling this - we should update `Divider` to have the same name.

Reply:
accidently merged this change to main 😬